django/contrib/gis/geoip2/base.py

```diff
@@ -92,6 +92,9 @@ def __init__(self, path=None, cache=0, country=None, city=None):
             if os.path.isfile(city_db):
                 self._city = geoip2.database.Reader(city_db, mode=cache)
                 self._city_file = city_db
+
+            if not self._country and not self._city:
+                raise GeoIP2Exception('Path %s must contains a valid database file.' % path)
```

Comment:
Minor spelling tweak - "must contain a", not "must contains a".

Reply:
Thank you for noticing, fixed now.